src/test/index.test.ts

```diff
@@ -88,6 +91,49 @@ describe("Parse with custom converter functions", () => {
   });
 });
 
+describe("Parse unsupported tags", () => {
+  it("Skips element by default", () => {
+    const matchText = "This is text in a custom tag";
+    const htmlNodes = htmlStringToDocument(
+      `<p><custom-tag>${matchText}</custom-tag></p>`,
+    );
+
+    const matchNode = createDocumentNode([
+      helpers.createBlock(BLOCKS.PARAGRAPH, helpers.createText(matchText)),
+    ] as TopLevelBlock[]);
+
+    expect(htmlNodes).toMatchObject(matchNode);
+    expect(validateRichTextDocument(htmlNodes).length).toEqual(0);
+  });
+
+  it("Uses default converter when specified", () => {
+    const defaultConverter: TagConverter<Block> = (node, next) => {
+      return {
+        nodeType: BLOCKS.PARAGRAPH,
+        content: next(node),
+        data: {},
+      };
+    };
+    const matchText = "This is text in a custom tag";
+    const htmlNodes = htmlStringToDocument(
+      `<custom-tag>${matchText}</custom-tag>`,
+      {
+        convertTag: {
+          [DEFAULT_TAG_CONVERTER]: defaultConverter,
+        },
+      },
+    );
+
+    const matchNode = createDocumentNode([
+      helpers.createBlock(BLOCKS.PARAGRAPH, helpers.createText(matchText)),
+    ] as TopLevelBlock[]);
+
+    expect(htmlNodes).toMatchObject(matchNode);
+    console.log(validateRichTextDocument(htmlNodes));
```

Comment:
Could we remove the `console.log` here?

Reply:
Sure! See https://github.com/oleast/contentful-rich-text-html-parser/pull/256/commits/bd710f2b1165a376f3f9a66c7ab913c191954ea7.